Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1920,7 +1923,7 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         discoveryGroup.async(queue: discoveryQueue) { [unowned self] in
             symbolGraphLoader = SymbolGraphLoader(bundle: bundle, dataProvider: self.dataProvider)
             do {
-                try symbolGraphLoader.loadAll()
+                try symbolGraphLoader.loadAll(using: decoder)
```

Comment:
Mostly out of curiosity: why is a decoded passed as an argument? AFIACT the context's decoder is never modified.

Reply:
This can be used (in tests) to decode symbol kinds not known to SymbolKit. See e.g. the changes in `Tests/SwiftDocCTests/Infrastructure/AutomaticCurationTests.swift`.

This registration process is part of apple/swift-docc-symbolkit#39. For usage details please refer to [the documentation here](https://github.com/apple/swift-docc-symbolkit/blob/68452289f26e6990b78e185bdfeb855a6a7c5e1e/Sources/SymbolKit/SymbolKit.docc/GraphFormatExtensions.md).